options/ansi/generic/ctype-stubs.cpp

```diff
@@ -254,9 +254,47 @@ wctype_t wctype(const char *cs) {
 	return ct_null;
 }
 
-int iswctype(wint_t, wctype_t) {
-	__ensure(!"Not implemented");
-	__builtin_unreachable();
+#define WCTYPE_ALNUM 1
```

Comment:
Where do these defines come from?

Reply:
Oops I didn't see our definitions above, fixed it to use those.